netlify.toml

```diff
@@ -1,3 +1,9 @@
 [build]
   publish = "_site/"
   command = "npm run prod && jekyll build"
+
+[[redirects]]
+  # Previously, Doxygen docs were hosted under "m.vtk.org/documentation/",
```

Comment:
@brianhelba  here to:  https://docs-m.vtk.org/

Reply:
I'm not sure what you mean. The old links were indeed under `/documentation/`.

Reply:
my bad, I went to fast into correcting the url. ignore this please